Source/dom/export.js

```diff
@@ -46,58 +46,146 @@ export const DEFAULT_EXPORT_OPTIONS = {
  *
  *
  * @param {dictionary} options Options indicating which sizes and formats to use, etc.
+ *
+ * @returns If an output path is not set, the data is returned
  */
 export function exportObject(object, options) {
-  const merged = { ...DEFAULT_EXPORT_OPTIONS, ...options }
+  if (!object) {
+    throw new Error('No object provided to export')
+  }
+  let exportJSON = false
+  let { formats } = options
```

Comment:
I believe you need `options || {}` otherwise it will throw when there is no options

```suggestion
  let { formats } = options || {}
```